src/pages/index.module.css

```diff
@@ -1,23 +1,405 @@
-/**
- * CSS files with the .module.css suffix will be treated as CSS modules
- * and scoped locally.
- */
+/* ----- Elements ------  */
 
-.heroBanner {
-	padding: 4rem 0;
+:root {
+	--pink: #f37;
+	--blue: #2563eb;
+	--bg-blue: #e0f2fe;
+}
+
+.main code {
+	font-size: 0.8rem !important;
+}
+
+.main {
+	margin: 1rem;
+}
+
+.main > section {
+	max-width: 50rem;
+	margin: 0 auto;
+	margin-top: 5rem;
+}
+
+.main section h2,
+.main section p {
+	max-width: 30rem;
+}
+
+.button {
+	color: var(--blue);
+	font-weight: 600;
+	background-color: var(--bg-blue);
+	padding: 0.5rem 1rem;
+	border-radius: 5px;
+	transition: filter 150ms ease-in-out;
+}
+
+.button:hover,
+.button:focus {
+	text-decoration: none;
+	color: var(--blue);
+	filter: brightness(95%);
+}
+
+/* ----- Intro ------  */
+
+.header {
+	margin: 1rem auto;
+	padding: 0 1rem;
+	margin-top: 7rem;
 	text-align: center;
+	max-width: 30rem;
+}
+@media screen and (min-width: 40rem) {
+	.header {
+		padding-top: 5rem;
+		min-height: 75vh;
+	}
+}
+
+.quote:first-of-type {
+	margin-top: 2rem;
+}
+
+.quote blockquote {
+	margin: 0;
+	padding: 0;
+	border: none;
+	font-style: italic;
+}
+
+.quote figcaption a {
+	font-size: 0.85rem;
+	color: #6b7280;
+	text-decoration: underline;
+}
+
+/* ----- Dashboard ------  */
+
+.dashboard {
 	position: relative;
-	overflow: hidden;
 }
 
-@media screen and (max-width: 996px) {
-	.heroBanner {
-		padding: 2rem;
+.main .section-dashboard {
+	max-width: 65rem;
+}
+
+.dashboard::after {
+	position: absolute;
+	content: '';
+	top: 0;
+	right: 0;
+	z-index: -1;
+	height: 100%;
+	width: 100%;
+	margin: 0 auto;
+	filter: blur(60px);
+	background: linear-gradient(90deg, rgb(255, 51, 119, 0.4), #fbcfe8);
+	background-size: 200% 200%;
+	background-position: 100% 50%;
+}
+
+/* Don't show code example on mobile, it's too big */
+.dashboard div[class*='codeBlockContainer'] {
+	display: none;
+}
+@media screen and (min-width: 40rem) {
+	.dashboard div[class*='codeBlockContainer'] {
+		display: block;
+	}
+}
+@media screen and (min-width: 50rem) {
+	.dashboard {
+		display: grid;
+		grid-template-columns: repeat(2, minmax(0, 1fr));
+		align-items: center;
+	}
+	.dashboard::after {
+		width: 300px;
+		right: calc(50% - 300px / 2);
+	}
+}
+
+.upload {
+	position: absolute;
+	z-index: -10;
+	content: '';
+}
+
+.upload-one {
+	transform: rotate(5deg) translateY(30px);
+	top: 0;
+	right: 50px;
+	height: 130%;
+	width: 150px;
+}
+
+.upload-two {
+	transform: rotate(-10deg) translateY(30px);
+	top: 0;
+	left: 400px;
+	height: 130%;
+	width: 150px;
+	display: none;
+}
+
+.upload svg {
+	transform: rotate(7deg) translateY(55px) translateX(-10px);
+	width: 100%;
+}
+
+.upload div {
```

Comment:
I would add an extra css class vs targeting a div.

Reply:
I'm not too worried here because `upload` are the flying uploads, they are specific and decorative, not reusable components. Also because nothing of the homepage CSS will be used anywhere but on this page.